statham/schema/elements/meta.py

```diff
@@ -170,6 +172,10 @@ def python(cls) -> str:
             cls_args.append(f"{param.name}={repr(value)}")
         class_def = f"""class {repr(cls)}({', '.join(cls_args)}):
 """
+        if not cls.description is None and not isinstance(
+            cls.description, NotPassed
+        ):
+            class_def += f'\n    """{cls.description}"""\n'
```

Comment:
```suggestion
            class_def += f'    """{cls.description}"""\n'
```

Minor nit: prefer

```python
class MyClass:
    """Docstring."""
```

over

```python
class MyClass:

    """Docstring."""
```